dll/cpl/desk/lang/pt-PT.rc

```diff
@@ -40,7 +42,7 @@ CAPTION "Protecção de ecrã"
 FONT 8, "MS Shell Dlg", 0, 0
 BEGIN
     CONTROL "", IDC_SCREENS_PREVIEW, "Static", SS_OWNERDRAW, 70, 10, 105, 74
-    GROUPBOX "Protecção de &ecrã", IDC_SCREENS_DUMMY, 8, 92, 230, 64
+    GROUPBOX "Pro&tecção de ecrã", IDC_SCREENS_DUMMY, 8, 92, 230, 64
```

Comment:
Yes, better than before with the collision. But since I notidced recently that &t doesn't look very good (very narrow character) I do wonder whether MS uses &t here as well, or something else. Can you prove with a pt-PT MS-pic?

Reply:
![VirtualBox_WinSrv2003_Monitor](https://github.com/reactos/reactos/assets/49964946/67e7d46c-c39c-434d-bccc-94bd34118879)
There he is.

Reply:
MS is king. Approved then. Thx.